brooklyn-server/rest/rest-server/src/main/java/org/apache/brooklyn/rest/util/json/BrooklynJacksonJsonProvider.java

```diff
@@ -137,25 +136,21 @@ public static ObjectMapper newPrivateObjectMapper(ManagementContext mgmt) {
             throw new IllegalStateException("No management context available for creating ObjectMapper");
         }
 
-        SerializationConfig defaultConfig = new ObjectMapper().getSerializationConfig();
-        SerializationConfig sc = new SerializationConfig(
-            defaultConfig.getClassIntrospector() /* ObjectMapper.DEFAULT_INTROSPECTOR */,
-            defaultConfig.getAnnotationIntrospector() /* ObjectMapper.DEFAULT_ANNOTATION_INTROSPECTOR */,
-            new PossiblyStrictPreferringFieldsVisibilityChecker(),
-            null, null, TypeFactory.defaultInstance(), null);
-
         ConfigurableSerializerProvider sp = new ConfigurableSerializerProvider();
         sp.setUnknownTypeSerializer(new ErrorAndToStringUnknownTypeSerializer());
 
-        ObjectMapper mapper = new ObjectMapper(null, sp, null, sc, null);
+        ObjectMapper mapper = new ObjectMapper();
+        mapper.setSerializerProvider(sp);
+        mapper.setVisibility(new PossiblyStrictPreferringFieldsVisibilityChecker());
+
         SimpleModule mapperModule = new SimpleModule("Brooklyn", new Version(0, 0, 0, "ignored"));
 
         new BidiSerialization.ManagementContextSerialization(mgmt).install(mapperModule);
         new BidiSerialization.EntitySerialization(mgmt).install(mapperModule);
         new BidiSerialization.LocationSerialization(mgmt).install(mapperModule);
-        mapperModule.addSerializer(new MultimapSerializer());
```

Comment:
I've followed this [suggestion](https://github.com/apache/incubator-brooklyn/blob/9d5ba14b91424b77e48468518fe74256b12a8af8/brooklyn-server/rest/rest-server/src/main/java/org/apache/brooklyn/rest/util/json/MultimapSerializer.java#L36-L38) 
`GuavaModule` from `jackson-datatype-guava` should be taking care of Multimap serialization among others.